helper_functions/EXTRACT_output_to_nwb.m

```diff
@@ -0,0 +1,150 @@
+function nwb = EXTRACT_output_to_nwb(output, options)
+% EXTRACT_OUTPUT_TO_NWB writes the data within the EXTRACT output structure
+% to an NWB file
+%
+%  output: EXTRACT output structure
+%
+%  options: structure with details necessary for nwb file creation.
```

Comment:
It seems a key, value pair syntax might work better here.

Reply:
Look into [inputParser](https://www.mathworks.com/help/matlab/ref/inputparser.html)

Reply:
or were you doing it this way to try to copy the style of EXTRACT?